scripts/dap-ami-bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
```

Comment:
Do we need to wait for docker to be available before running `docker images` ? It can lead to grep on "Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?"

And is the searching for appliance image necessery at all? If it's bootstrap there should not be any images in - and if there is it will have the same sha and would be quickly load into docker cache

Reply:
> Do we need to wait for docker to be available before running docker images
SystemD should be waiting for Docker to be available before returning. This hasn't caused any issues in the field or in tests as far as I'm aware.

> And is the searching for appliance image necessery at all?
As of this PR the script can be re-run, and the image will be missing from the registry/available from /tmp only on the first run.